phpstan.neon.dist

```diff
@@ -1,9 +1,28 @@
 includes:
     - ./vendor/larastan/larastan/extension.neon
 parameters:
-    level: 1
+    level: 2
     paths:
         - src
         - tests
     ignoreErrors:
         - '#Unsafe usage of new static\(\).#'
+        - '#Access to an undefined property Staudenmeir\\LaravelAdjacencyList\\Tests\\Tree\\Models\\[a-zA-Z\\]+::\$[a-zA-Z0-9_]+#'
+        - '#Access to an undefined property Illuminate\\Database\\Query\\Builder::\$expressions#'
+        - '#Access to an undefined property Illuminate\\Database\\Eloquent\\Model::\$laravel_through_key#'
+        - '#Call to an undefined method TRelatedModel of Illuminate\\Database\\Eloquent\\Model#'
+        - '#Call to an undefined method TDeclaringModel of Illuminate\\Database\\Eloquent\\Model#'
+        - '#Call to an undefined method TModel of Illuminate\\Database\\Eloquent\\Model#'
+        - '#Call to an undefined method Illuminate\\Database\\Eloquent\\Model::#'
+        - '#Call to an undefined method Illuminate\\Database\\Eloquent\\Builder#'
+        - '#Call to an undefined method Illuminate\\Database\\Schema\\Blueprint::shardKey\(\)#'
+        - '#Call to an undefined method Illuminate\\Database\\ConnectionInterface#'
+        - '#Call to an undefined method Staudenmeir\\LaravelAdjacencyList\\Query\\Grammars\\ExpressionGrammar::wrap\(\)#'
+        - '#PHPDoc tag \@param for parameter \$grammar with type Illuminate\\Database\\Grammar\|Staudenmeir\\LaravelAdjacencyList\\Query\\Grammars\\ExpressionGrammar is not subtype#'
```

Comment:
would require adding ExpressionGrammar as a native type, but that could be a breaking change

Reply:
I resolved this by removing `|\Illuminate\Database\Grammar` from the `@param` comment and adding the `wrap()` method to the `ExpressionGrammar` interface. This also resolves the `Call to an undefined method Staudenmeir\\LaravelAdjacencyList\\Query\\Grammars\\ExpressionGrammar::wrap\(\)` error.